nannou_wgpu/src/texture/capturer.rs

```diff
@@ -61,49 +53,10 @@ struct ConverterDataPair {
 pub struct Rgba8AsyncMappedImageBuffer<'buffer>(wgpu::ImageReadMapping<'buffer>);
 
 impl ThreadPool {
-    /// Spawns the given future if a worker is available. Otherwise, blocks and waits for a worker
-    /// to become available before spawning the future.
-    fn spawn_when_worker_available<F>(&self, future: F) -> Result<(), AwaitWorkerTimeout<F>>
```

Comment:
I have no idea whether this code is important or not. It looks to me as if something is done manually that should be handled by a thread pool executor.

Reply:
Okay... I see this breaks more things...